site/src/components/input.md

```diff
@@ -45,6 +45,19 @@ Here is an example of a standard usage of input paired with a label.
 | Danger  | `.input .input-danger`  | <input class="input input-danger" type="text" placeholder="Placeholder" />  |
 | Success | `.input .input-success` | <input class="input input-success" type="text" placeholder="Placeholder" /> |
 
-```abut-html
-<input class="input input-danger" type="text" placeholder="Placeholder" />
+```html
+<input class="input input-danger" type="text" />
+```
+
+### Modifiers
+
+Use `input-icon` on the input in combination with a `.icon` sibling to add an icon to the input. If the icon needs to displayed at the end of the input, use `input-icon-right`.
+
+```html
+<div class="position-relative margin-bottom-xxs">
+	<input class="input input-icon" type="text" /> <span class="icon border"></span>
```

Comment:
If possible, would love to see an example with an inlined svg here as well, since that is how we do this on the site. I was messing in the dev tools with this and wasn't able to get the result I wanted - but that could be paternity leave brain 😆 ...

Reply:
I ended up directly taking the svg from docons to show the icon.